CMakeLists.txt

```diff
@@ -109,6 +109,9 @@ cmake_dependent_option(BUILD_NVOF "Build with NVIDIA OPTICAL FLOW SDK support" O
                        "NOT BUILD_DALI_NODEPS" OFF)
 cmake_dependent_option(BUILD_NVML "Build with NVIDIA Management Library (NVML) support" ON
                        "NOT BUILD_DALI_NODEPS" OFF)
+
+cmake_dependent_option(BUILD_NVCOMP "Build with nvCOMP support" OFF
+                       "NOT BUILD_DALI_NODEPS" OFF)
```

Comment:
Shouldn't it be `ON` by default?

Reply:
Well, in case of NVJPEG2k you can set it to on and it will disable itself if the library isn't found.

Reply:
For example for cufile we have OFF by default. I am not sure if overriding the option silently rather than explicitly failing if we cannot build requested conf is a good way to go.

Reply:
OK, it's not a blocker anyway.